ocs_ci/deployment/deployment.py

```diff
@@ -1537,6 +1542,93 @@ def deploy_ocs_via_operator(self, image=None):
                 f"{constants.NODE_SELECTOR_ANNOTATION}"
             )
 
+        # Access buckets with DNS subdomain style (Virtual host style) for RGW
+        self.configure_virtual_host_style_acess_for_rgw()
```

Comment:
Here we need to think of also if deployment is done via UI, then this code will be skipped.

Thinking about moving it somewhere here:
https://github.com/dahorak/ocs-ci/blob/10ff4dd9ca8e650a3ee7188d253a9b065ee70cce/ocs_ci/deployment/deployment.py#L1950

Which could handle post UI and CLI deployment cases

Reply:
Updated here: [b0a8bf1](https://github.com/red-hat-storage/ocs-ci/pull/9839/commits/b0a8bf1d3974ebc9368578f75707ddffa3fd71db)